tasks/db_deploy/orca_sql.py

```diff
@@ -168,8 +168,15 @@ def app_user_sql(user_password: str) -> TextClause:
         user_password (str): Password for the application user
 
     Returns:
-        (sqlalchemy.sql.element.TextClause): SQL for creating orcauser user.
+        (sqlalchemy.sql.element.TextClause): SQL for creating PREFIX_orcauser user.
     """
+    if user_name is None or len(user_name) == 0:
+        logger.critical("Username must be non-empty.")
+        raise Exception("Username not long enough.")
```

Comment:
I would make the exception message the same as the log message.

Reply:
Unified messages